src/gicv3.rs

```diff
@@ -20,6 +20,39 @@ use core::{
 /// The offset in bytes from `RD_base` to `SGI_base`.
 const SGI_OFFSET: usize = 0x10000;
 
+enum BitOp {
```

Comment:
I'm not sure it really adds much to use an enum for this rather than a boolean. We really are just setting the bit to true or false, right?